editor/src/messages/tool/transform_layer/transform_layer_message_handler.rs

```diff
@@ -120,6 +148,56 @@ impl MessageHandler<TransformLayerMessage, TransformData<'_>> for TransformLayer
 				selected.original_transforms.clear();
 			}
 			TransformLayerMessage::BeginRotate => {
+				if !(using_path_tool || using_select_tool) {
+					return;
+				}
+				let Some(&layer) = selected_layers.first() else {
+					return;
+				};
+				let Some(vector_data) = document.network_interface.compute_modified_vector(layer) else {
+					return;
+				};
+				let points = shape_editor.selected_points();
+				let selected_points: Vec<&ManipulatorPointId> = points.collect();
+
+				if using_path_tool && selected_points.is_empty() {
+					responses.add(TransformLayerMessage::CancelTransformOperation);
+					return;
+				}
+
+				if selected_points.len() == 1 {
+					if let Some(point) = selected_points.first() {
+						match point {
+							ManipulatorPointId::Anchor(_) => {
+								if let Some([handle1, handle2]) = point.get_handle_pair(&vector_data) {
+									if (handle1.get_handle_length(&vector_data) == 0.0 && handle2.get_handle_length(&vector_data) == 0.0)
```

Comment:
Please avoid using the style `0.0` and use the `0.` style instead with decimals.